locale/en/docs/guides/simple-profiling.md

```diff
@@ -86,10 +97,15 @@ high latency on requests. We can easily run the app with the built in profiler:
 NODE_ENV=production node --prof app.js
 ```
 
-and put some load on the server using `ab` (ApacheBench):
+We'll create a new user:
 
 ```
 curl -X GET "http://localhost:8080/newUser?username=matt&password=password"
```

Comment:
I think we should not be recommending to create a new user via a GET.

Reply:
Agreed,  but I think this should be included with the suggestion to replace ab with auto cannon, since that's going to require an update to the output and interpretation of the data.